pkg/secret/secret.go

```diff
@@ -65,7 +65,8 @@ func (o *Secret) Save() (err error) {
 		obj, err = o.client().Create(o.SecretApi)
 	}
 	if err != nil {
-		return
+		o.Log().Fatalf("error creating/updating secret: %v", err.Error())
```

Comment:
Fatalf it's not a good point here, maybe the api are unreachable during this time why crash kube-lego?